core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/AggregatedWorkspaceService.scala

```diff
@@ -43,14 +44,14 @@ class AggregatedWorkspaceService(workspaceManagerDAO: WorkspaceManagerDAO) exten
     val span = startSpanWithParent("getWorkspaceFromWorkspaceManager", ctx.tracingSpan.orNull)
     try {
       val wsmInfo = workspaceManagerDAO.getWorkspace(workspace.workspaceIdAsUUID, ctx)
-      (wsmInfo.getStage, Option(wsmInfo.getGcpContext), Option(wsmInfo.getAzureContext)) match {
-        case (WorkspaceStageModel.RAWLS_WORKSPACE, _, _) =>
+      (wsmInfo.getStage, Option(wsmInfo.getGcpContext), Option(wsmInfo.getAzureContext), workspace.state) match {
```

Comment:
Oof, this got pretty intense (scala newb here).  I defer to you on this as the code owner, but definitely whistled past this block on my way to the stuff I'm more familiar with.